### ISSUE TITLE
Standard configuration is broken for humidifier

### ISSUE THREAD

**Issue by lmagyar** [nielsfaber/scheduler-card#404]
Scheduler v3.1.2
Card v2.1.3 (browser cache deleted)
HA 2021.5.4 OS

I'm aware of #76, #82, #210, that this is in a "semi public" state.

I'm using home-assistant/core#36759 (yeah, generic_hygrostat is still not merged).

cc: @KTibow from #82

Problem: even if I add

```yaml
type: 'custom:scheduler-card'
include:
  - humidifier
```

or

```yaml
type: 'custom:scheduler-card'
include:
  - humidifier.some_concrete_entity
```

Nothing shows up when I try to add a new schedule:

![image](https://user-images.githubusercontent.com/5061219/118375040-73233400-b5bf-11eb-82d6-7205f36acc58.png)

I have to add the actions also:

```yaml
include:
  - humidifier
customize:
  humidifier:
    actions:
      - service: set_humidity
        icon: 'hass:water-percent'
        name: 'set humidity[ to {humidity}]'
        variables:
          humidity:
            name: Humidity
            unit: '%'
            min: 40
            max: 80
            step: 1
```

So I think something is broken, if the default settings/actions are there, but not working.

**Comment by lmagyar** [comment]
I'm using only `climate` and `humidifier` domain, but climate works as expected.

**Comment by nielsfaber** [comment]
I found a typo in my code, it was looking for `humidifer` entities instead...
Fix is on the way: 
https://github.com/nielsfaber/scheduler-card/commit/ab6afe0dd44c52e76185c02cb746c8ac4f799fba#diff-fcece7569bcbae5e2c4bf12818c011d755d52aa9f91bb4f7ebf2dd198b0bf0f0

**Comment by lmagyar** [comment]
It is fixed in v2.1.4.
